scripts/appName.js

```diff
@@ -4,8 +4,40 @@ const getTrimString = (string) => string.split(' ').join('')
 const getLowerCaseString = (string) => string.toLowerCase()
 const getLowerCasedUnderscoredString = (string) => getTrimString(getLowerCaseString(string))
 
-module.exports.appName = configuration.applicationName
+const getApplicationName = () => {
+  try {
+    if (!configuration || configuration === undefined) {
+      throw new Error('Not configuration provided')
+    }
 
-module.exports.shortName = configuration.shortName
-  ? configuration.shortName
-  : getLowerCasedUnderscoredString(configuration.applicationName)
+    if (!configuration.applicationName || configuration.applicationName === '') {
+      throw new Error('Application name is a required parameter, please provide it')
+    }
+
+    return configuration.applicationName
+  } catch (error) {
+    throw new Error(error)
+  }
+}
+
+const getShortApplicationName = () => {
+  try {
```

Comment:
This name is used for setting the stack name, right? We should figure out what the maximum character limit we can have is (before CloudFormation starts throwing error for having a long IAM role name or whatever it is)